pkg/ctxutil/ctxutil.go

```diff
@@ -271,6 +322,50 @@ func GetCommitMessage(ctx context.Context) string {
 	return sv
 }
 
+// GetCommitMessageFull returns the set commit message (head+body, of either are defined) or an empty string.
+func GetCommitMessageFull(ctx context.Context) string {
+	sv_head, ok_head := ctx.Value(ctxKeyCommitMessage).(string)
+	sv_body, ok_body := ctx.Value(ctxKeyCommitMessageBody).(string)
+	if !(ok_body||ok_head) {
+		return ""
+	} else if !ok_body {
+		return sv_head
+	} else if !ok_head {
+		return sv_body
+	} else {
+		return sv_head + "\n\n" + sv_body
+	}
+}
+
+// ClearErrorCollector returns a context with where the error collector is emptied.
+func ClearErrorCollector(ctx context.Context) context.Context {
+	return context.WithValue(ctx, ctxKeyErrorCollector, nil)
+}
+// AddToErrorCollector returns a context with a new error logged as a new line into the error collector
+func AddToErrorCollector(ctx context.Context, sv string) context.Context {
```

Comment:
My remarks about the commit message builder also apply here.
Instead of concatenating strings we might create (or use an existing) multi error type that is referenced here.

But: do we really need to store this in the context?
If there other options are too cumbersome that's find, but I do consider using the context as a last restort. It's not very elegant and we've been trying to recude the use of the context in the past already.

Reply:
that part of the code in ctxutil.go was unused anyway, so I just removed it.
(I originally added it before fsck was moved to `WithQueue(nil)`)